_docs/crowfacts.md

```diff
@@ -223,7 +229,22 @@ the custom errors can be seen
+    
+#### Set up environment variables for Lambda functions
+- In order to keep the code looking clean, and to keep our Topic ARNs safe (more on these in the DynamoDB section), we use environment variables to store the names of the DynamoDB topics and their ARNs. These are exactly like environment variables on your computer. They're variables that you can access in your code like any other, but since they're not declared or otherwise modified in the source code, they're invisible and can't be accidentally leaked (say when you upload your code to a public git repository). 
+
+- After you create a Lambda function in the AWS portal, you will be greeted with the function's page. Here you can edit the code directly and modify the function's settings. Right below the code window, you will see a menu item for **Environment Variables**. Click Edit to modify them. The **key** is the variable name, and the **value** is the variable's value. Click Save to save these variables in the function. Now you can access the environment variables in the Lambda function's code by calling it using the key. See [AWS Documentation on Environment Variables](https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html) for more info.
+
+#### Set up aliases for Lambda functions
+- Lambda has a feature called **Aliases**. Aliases are used in combination with function versions to manage what version of the Lambda function will be called by other AWS services. In CrowFacts, aliases are used to connect API Gateway endpoints to the publish and subscribe Lambda functions. 
+
+- In order to create an alias, you must first have a **published function version** to connect it to. Under the Actions menu in the function, click on **Publish new version**, enter a description, and publish it to create a new version. This new version will retain the functionality that it has at the time that it is published. Refer to the [AWS Docs on Lambda Versions](https://docs.aws.amazon.com/lambda/latest/dg/configuration-versions.html) for more detail on what you can do with versioning.
+
+- **Aliases allow you to refer to different versions of a function with the same name.** So for example in API Gateway you can link a GET request with a Lambda function using the alias "API_GET_Version". Then, no matter what function version you connect to that alias, be it 1, 10, or 1000, you will not need to change the API Gateway configuration when you update the Lambda code. You do however, need to modify which function version the alias will point to each time you publish a new version. You *can* alias to the `$LATEST` version in order to have Lambda automatically use whatever code exists when you Save the function, but this is not recommended. You should only alias to function versions that you have tested and verified to work. Further documentation on function aliases is available [here](https://docs.aws.amazon.com/lambda/latest/dg/configuration-aliases.html).
```

Comment:
Removing this because most of our functions don't utilize aliasing in a meaningful way. There is no more APIVersion afaik, plus this is highly redundant with the FeelGood writeup.

```suggestion
```